methods/userEvents.coffee

```diff
@@ -61,3 +61,8 @@ Meteor.methods
         $unset:
           lastIncidentDate: ""
       })
+
+if Meteor.isServer
```

Comment:
I think this would be a better fit for server/interfaces.coffee since it is a server only method.